budgetiser/lib/shared/picker/select_icon.dart

```diff
@@ -48,25 +65,35 @@ class _IconPickerState extends State<IconPicker> {
   }
 
   Widget getIcons() {
-    return GridView.count(
-      physics: const ScrollPhysics(),
-      shrinkWrap: true,
-      crossAxisCount: 5,
-      children: List.generate(_icons.length, (index) {
-        return InkWell(
-          onTap: () {
-            setState(() {
-              _currentIcon = _icons[index];
-              widget.onIconChangedCallback(_icons[index]);
-              Navigator.pop(context);
-            });
-          },
-          child: Icon(
-            _icons[index],
-            color: widget.color,
-          ),
-        );
-      }),
+    List<IconData> filteredIcons = [];
+    for (var element in _fullIconList) {
+      if (element["name"].toLowerCase().contains(_searchString ?? "")) {
```

Comment:
wieso toLowerCase und nicht im Namen direkt alles klein schreiben in der Definition? Und was passiert wenn der User mit Auto Großschreibung einen Namen eintippt?

Reply:
habs angepasst